src/audio/dai.c

```diff
@@ -393,7 +393,9 @@ static void dai_data_config(struct comp_dev *dev)
 		dd->stream_id = dd->dai_config->alh.stream_id;
```

Comment:
`src` prefix is redundant in the commit message. Please use `git log src/audio/dai.c` and see what other have used for the prefixes.

Also, the commit subject doesn't make much sense. Please make the commit message to be an action that this commit is doing on the existing code.

e.g dai: imx: Add support for i.MX8ULP

Also, please read again https://chris.beams.io/posts/git-commit/#seven-rules